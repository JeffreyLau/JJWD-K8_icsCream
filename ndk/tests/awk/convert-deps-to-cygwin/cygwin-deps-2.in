D:/Stuff/source.o: D:/Stuff/source.h C:/NDK/sysroot/include/string.h \
 C:/NDK/sysroot/include/malloc.h
