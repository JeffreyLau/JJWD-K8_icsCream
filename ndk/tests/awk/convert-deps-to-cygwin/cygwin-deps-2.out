/cygdrive/d/Stuff/source.o: /cygdrive/d/Stuff/source.h /cygdrive/c/NDK/sysroot/include/string.h \
 /cygdrive/c/NDK/sysroot/include/malloc.h
